### ISSUE TITLE
Where can I get the DJI windows sdk.

### ISSUE THREAD

**Issue by mohansriramakrishna** [dji-sdk/DJI-Windows-SDK-UWP-Sample#2]
Anyone please give me link to the windows sdk.

**Comment by giggio** [comment]
The link is this: https://developer.dji.com/windows-sdk/
But it says "Coming soon".

**Comment by giggio** [comment]
@mariankh see this comment for now: https://github.com/dji-sdk/DJI-Windows-SDK-UWP-Sample/issues/1#issuecomment-412973707

**Comment by mariankh** [comment]
Thank you @giggio!

**Comment by mariankh** [comment]
I am missing the DJIWindowsSDKDemo/ThirdParties .. anywhere i can find it?

**Comment by jimmyfraitureUCL** [comment]
Did you find the ThirdParties?

**Comment by mariankh** [comment]
Nop! Still looking for it !

**Comment by jimmyfraitureUCL** [comment]
So impossible to use the sdk for the moment?

**Comment by snakehaihai** [comment]
only a few invited party can access full dji windows sdk.  u might go and request from some party who have access to it. 
![image](https://user-images.githubusercontent.com/6070995/47689075-3242ba00-dc23-11e8-9571-e7e4d9651596.png)


and to be honest its still quite alpha nothing much implemented yet. 

![image](https://user-images.githubusercontent.com/6070995/47689034-058ea280-dc23-11e8-9bea-c68c92b10a37.png)

**Comment by XietongLU** [comment]
The Public Alpha version will be released very very very soon!

**Comment by giggio** [comment]
It is up now: https://developer.dji.com/windows-sdk/downloads/

**Comment by rhoegen** [comment]
Did anyone have success getting this one compiled? I am still missing a few header files (sdkmanager.h, videofeed.h, videofeeder.h and virtualremotecontroller.h) and I had no success getting them from any DJI resources.

**Comment by HerrickSpencer** [comment]
The SDK was made available at this location in the DJI Developer site.
https://developer.dji.com/windows-sdk/
 
rhoegen, the headers you are looking for are in that SDK

**Comment by CodeNeedsCoffee** [comment]
@HerrickSpencer  I just downloaded the SDK from https://developer.dji.com/windows-sdk/ and I can't find those header files in it.
@rhoegen did you find those header files? 

Any help is appreciated!
Cheers!

**Comment by HerrickSpencer** [comment]
You are correct, I was using a previous release.  I will alert the DJI team and try to get this resolved for us (because I use it also).
